api/v1alpha3/types.go

```diff
@@ -60,8 +60,15 @@ type VnetSpec struct {
 	Name string `json:"name"`
 
 	// CidrBlock is the CIDR block to be used when the provider creates a managed virtual network.
+	// TODO (#618) should just make this a list as is done in https://docs.microsoft.com/en-us/cli/azure/network/vnet?view=azure-cli-latest#az-network-vnet-create
 	CidrBlock string `json:"cidrBlock,omitempty"`
 
+	// Ipv6CidrBlock is the cidr block if IPv6 is is enabled
+	IPv6CidrBlock string `json:"iPv6CidrBlock,omitempty"`
+
+	// EnableIPv6 turns IPv6 on
+	IPv6Enabled bool `json:"iPv6Enabled,omitempty"`
```

Comment:
Thoughts on how to represent dual stack vs. IPv6 only in the future?

Reply:
For aks-engine, we have 2 flags - `EnableDualStack` and `EnableIPv6`. IPv6 on azure is just a subset of dual-stack in terms of the infra bootstrapping. We support IPv6 on dual-stack hosts. I think in the future, when we add support for dual-stack clusters, we would need to add a new flag `DualstackEnabled` which would be used to configure the Kubernetes specific params required for dual-stack clusters. For infra, we would have the same code path for IPv6/Dualstack. WDYT?